docs/pages/distribution/release-channels.md

```diff
@@ -19,6 +19,8 @@ Publish your release by running:
 
 with the Expo CLI. Your users can see this release in the Expo Go app with a parameterized URL `https://exp.host/@username/yourApp?release-channel=<your-channel>`. If you do not specify a channel, you will publish to the `default` channel.
 
+A release channel name can only contain lowercase letters, numbers and special characters `.`, `_` and `-`.
```

Comment:
Is this actually true though? https://github.com/expo/eas-cli/issues/955

Reply:
Ah, I think that is for eas *channels* while this is for the classic *releaseChannels*.

Here is where the validation is enforced server side: https://github.com/expo/universe/blob/d7bd95d426fbc1fd708e23a6aae2f7ea2917550a/server/www/src/apiv2/PublishRouter.ts#L134

Reply:
I believe he was referring to classic updates but I asked for clarification https://github.com/expo/eas-cli/issues/955#issuecomment-1028492209

Reply:
classic `expo publish` confirmed in that issue.

Reply:
gotcha. @jkhales it looks like this needs more investigation for release channels then

Reply:
Yea true, I'll take a look at the eas build integration. But I think *this* PR is correct. Or am I missing the point? 😅

Reply:
If there's an issue, either fix the issue, or fix the docs. 😛

Reply:
@jkhales - this seems fine for this pr but it may be incorrect if it's not fixed in the EAS Build integration, so maybe we should follow up on that and then land this once that's fixed.

Reply:
Will do!